src/components/shared/index.ts

```diff
@@ -2,7 +2,6 @@ export * from './Button';
 export * from './ButtonGroup';
 export * from './Card';
 export * from './DropDown';
-export * from './modals/Modal';
```

Comment:
export this file again @ISHIMWE-Aime, i don't get why you are removing this file

Reply:
Thank you but I think you are mistaken, because I have just returned the file export at the moment you ask me why I have removed it. 
Thank you